src/plugins/intel_gpu/src/graph/graph_optimizer/mark_shape_of_subgraphs.cpp

```diff
@@ -86,13 +86,7 @@ bool mark_shape_of_subgraphs::can_mark_node(const program_node& node) {
         return false;
     }
 
-    auto available_impls = node.type()->get_available_impls(node);
-    auto cpu_impl_found = available_impls.find(impl_types::cpu) != available_impls.end();
-
-    if (cpu_impl_found)
-        return true;
-
-    return false;
```

Comment:
It will, but it shouldn't be an issue since markup shouldn't depend on the certain impl availability. Execution of those using CPU impls is just an optimization of execution time.